controllers/default_test.go

```diff
@@ -244,10 +244,10 @@ var _ = g.Describe("Reconcile default ShipwrightBuild installation", func() {
 			o.Expect(err).NotTo(o.HaveOccurred())
 
 			// Verify - check the behavior
-			test.CRDEventuallyRemoved(ctx, k8sClient, "builds.shipwright.io")
-			test.CRDEventuallyRemoved(ctx, k8sClient, "buildruns.shipwright.io")
-			test.CRDEventuallyRemoved(ctx, k8sClient, "buildstrategies.shipwright.io")
-			test.CRDEventuallyRemoved(ctx, k8sClient, "clusterbuildstrategies.shipwright.io")
+			test.CRDEventuallyExists(ctx, k8sClient, "builds.shipwright.io")
+			test.CRDEventuallyExists(ctx, k8sClient, "buildruns.shipwright.io")
+			test.CRDEventuallyExists(ctx, k8sClient, "buildstrategies.shipwright.io")
+			test.CRDEventuallyExists(ctx, k8sClient, "clusterbuildstrategies.shipwright.io")
```

Comment:
I wonder if we should introduce a wait period before checking that the CRDs still exist?

Reply:
@adambkaplan The test is passing without the wait period, we do the same for the deployment and rbac and those tests had been working since the begining.
https://github.com/shipwright-io/operator/blob/main/controllers/default_test.go#L222